chapters/en/slides/module2/module2_03.Rmd

```diff
@@ -0,0 +1,284 @@
+gm = pd.read_csv(url, parse_dates=['year'])
+gm2018 = gm[gm['year'] == '2018']
+gm2018
+```
+
+Notes:
+In this slide deck,
+we will continue to work with the gapminder dataset.
+To make it more suitable for this slide deck
+let's subset it to include observations from the year 2018 only.
+
+When filtering on a date in pandas,
+we can either type out just the year portion
+or the entire date string in the format shown in the slide.
+
+---
+
+## Creating a bar chart of the total population per continent
+
+```{python}
+import altair as alt
+
+
+alt.Chart(gm2018).mark_bar().encode(
+    x='region',
+    y='sum(population)')
+```
+
+Notes:
+Bar charts are often avoided when showing summary statistics such as the mean
+because they hide any variation,
+something we will talk more about in later modules.
+
+But they are fine for showing statistics
+such as counts of observations, summations, or individual values
+(e.g. the number of people in a country for a specific year).
+
+Here we create a bar chart of a single value per country,
+which represents the sum of all the countries' populations.
+
+---
+
+## Horizontal bar charts are made by switching the x and y encoding
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    y='region',
+    x='sum(population)')
+```
+
+Notes:
+If we switched x and y,
+we would create a horizontal bar chart instead.
+
+Although vertical bar charts are more commonly seen,
+horizontal bar charts are preferred
+when the labels on the categorical axis
+become so long that they need to be rotated
+to be readable in a vertical bar chart.
+
+Since this is the case for our plot
+we will continue to use horizontal bar charts
+in this slide deck.
+
+---
+
+## Visualizing counts of values with bar charts
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y='region')
+```
+
+Notes:
+To count values,
+we can use the aggregation function `count()` in Altair.
+We don't need to specify a column name for the y-axis,
+since we are just counting values in each categorical group on the y-axis.
+
+---
+
+## Ordering bars makes them easier to interpret
+
+```{python}
+alt.Chart(gm2018).mark_bar().encode(
+    x='count()',
+    y=alt.Y('region'))
+```
+
+Notes:
+Unless the categorical axis has a natural order to it,
+it is best to sort bar charts from high to low values.
+This makes it easier to see any trends in the data,
+and to compare bars of similar height more carefully.
+
+To sort the bars,
+we will use the helper functions `alt.X` and `alt.Y`,
+instead of `x=` and `y=`.
+These have the role of customizing things like order, groups, and scale for the axis.
+When using just `y='column'`,
+we're still calling `alt.Y()` under the hood,
+we just save ourselves some typing.
+
+In this slide,
+we use `alt.Y`
+on its own without any additional argument
+to recreate the same plot as above
+just to demonstrate how it works.
```

Comment:
I am not certain all this info is helpful on this slide... I feel like the purpose of this slide is to describe the problem, and the next is how to fix it.

Reply:
Moved much of this to the previous slide deck